utils/configuration.py

```diff
@@ -31,3 +33,6 @@ class Configuration:
 
     # Either 'ntsnet' (default), 'resnet256' or 'resnet128'
     network = 'ntsnet'
+
+    # This code only support meaning activation cluster
```

Comment:
Why do we need an entry in the config, if this is the only valid option?

Reply:
I have a plan to support both kinds of choosing the important cluster method, but actually not the case. If you are ok with only cluster using the meaning activation then I will remove that config